jupyrdf/ipyradiant
# SPARQLQueryFramer Testing

Adds unit tests for the `SPARQLQueryFramer` and a simple test to verify that metaclasses for dynamically defining the `VALUES` block of a SPARQL query (as part of the framer class) successfully execute (i.e. translate to the proper sparql string at execution time).

Test Procedure:
1. `doit`
2. `anaconda-project run test:pytest` (added this capability to support flags, e.g. `-k`)
3. Verify all tests pass, everything looks good, and green in CI
4. Review code

Closes #73
